### ISSUE TITLE
Release isoorbi 1.0.0

### ISSUE THREAD

**Issue by 123caj** [isoverse/isoorbiGUI#19]
First release:

* [x] `usethis::use_cran_comments()`
* [x] Update (aspirational) install instructions in README
* [x] Proofread `Title:` and `Description:`
* [ ] Check that all exported functions have `@return` and `@examples`
* [x] Check that `Authors@R:` includes a copyright holder (role 'cph')
* [x] Check [licensing of included files](https://r-pkgs.org/license.html#code-you-bundle)
* [ ] Review <https://github.com/DavisVaughan/extrachecks>

Prepare for release:

* [ ] `git pull`
* [ ] Check if any deprecation processes should be advanced, as described in [Gradual deprecation](https://lifecycle.r-lib.org/articles/communicate.html#gradual-deprecation)
* [ ] `urlchecker::url_check()`
* [ ] `devtools::check(remote = TRUE, manual = TRUE)`
* [ ] `devtools::check_win_devel()`
* [ ] `rhub::check_for_cran()`
* [ ] `git push`
* [ ] Draft blog post

Submit to CRAN:

* [ ] `usethis::use_version('major')`
* [ ] `devtools::submit_cran()`
* [ ] Approve email

Wait for CRAN...

* [ ] Accepted :tada:
* [ ] `git push`
* [ ] `usethis::use_github_release()`
* [ ] `usethis::use_dev_version()`
* [ ] `usethis::use_news_md()`
* [ ] `git push`
* [ ] Finish blog post
* [ ] Tweet
* [ ] Add link to blog post in pkgdown news menu

**Comment by 123caj** [comment]
- `devtools::check()` should return a single note max (that it's a new package), no errors, no warnings, no other notes

- once it passes the check, you can test it against the automated cran server with `devtools::check_win_release() `and then we can go from there

**Comment by sebkopf** [comment]
should this move to https://github.com/isoverse/isoorbi ?

**Comment by 123caj** [comment]
Yes and/or this issue can be closed, I think. This list was created with `usethis::use_release_issue()`
